### ISSUE TITLE
Resizing the gauges

### ISSUE THREAD

**Issue by PerLangkjaer** [colinbourassa/rovergauge#17]
Hello again

Hope you don't grow tired of me?
I'm now where I want to be and the next task is to make the bottom of the window smaller, thus making the top with the gauges a Little lager.
This should make it easier to read the gauges when driving instead of have a dash boad.
When the window is lager/maximized/fullscreen on my RPi it has some dead space under the text to the left and the need to read the fuelmap is not that big when driving around.
That is why I "just" want to move that line down.
I'm almost sure there are a good way to do it, just don't know how yet.

Hoping for some help
Per

![34517895-cc6f2f1e-f074-11e7-821a-7833f45fb73e](https://user-images.githubusercontent.com/35040883/34554463-4883a1d0-f12d-11e7-83be-ea7edccff906.png)

**Comment by PerLangkjaer** [comment]
This is a picture of it in maximized mode.
![2018-01-04-080111_1824x984_scrot](https://user-images.githubusercontent.com/35040883/34554601-8f145534-f125-11e7-93c3-322131382af2.png)

**Comment by PerLangkjaer** [comment]
After thinking about it, it would be best to leave it as is.
Then furthermore do a build that contains only the gauges and eventaully "Main Voltage" and "Selected Gear"

Would that be achiveable???

**Comment by cgalpin** [comment]
Yes, anything is doable - but you are going to have to learn a bit about QT and how to change the layout to do so. :)

**Comment by PerLangkjaer** [comment]
Doing the changes in the Qt Creator to the UI turns out to be pretty easy, got the design to where I want it, but trying to compile, install and run it, I got ALOT of errors.
Turns out that I cannot simply delete the stuff I don't want.

If one cound make the "line" I'm talking about in the first post, moveable it would be suffecient, and the it would have to start the program with the line pretty low, so that I'm would not have  to move it every time I got into the car.

As I'm sure you can understand, I'm no programmer, have never done any programming Before looking into this.

Learning Raspberry Pi, Linux, Qt, make, all on the same time like this turns out to be a mouth full.

**Comment by cgalpin** [comment]
You sound like you are doing just fine, especially for a non programmer.

I honestly don't have time to look into this and help or I would. I have run this on a small tablet and it would have been useful to do some resizing, including moving that line.

**Comment by PerLangkjaer** [comment]
I'll post here if I get a solution.
Maybe some help understanding what files could be deleted when it's running on a Rpi or better still an "apt-get" command.

**Comment by aclater** [comment]
Any progress on this - would love a gauge only view for smaller formfactor a-la RPi 4 3.5 inch screen

**Comment by PerLangkjaer** [comment]
@aclater 
Nope no solution and the car that it was going into was given up on some month ago.
